readme.md

```diff
@@ -1447,8 +1447,9 @@ const url = await moneyhub.getPaymentAuthorizeUrl({
   payeeId: "Id of payee", // required or payee
   payee: "Details of payee to create", // required or payeeId
   payeeType: "Payee type [api-payee|mh-user-account]", // optional - defaults to api-payee
-  payerId: "Id of payer", // required only if payerType is defined
-  payerType: "Payer type [mh-user-account]", // required only if payerId is used
+  payerType: "Payer type [mh-user-account | api-payer]", // required only if payerId or payer is used
```

Comment:
Would `api-payer` not get the details from the payees service?

Reply:
So we have added the new `api-payer` as a method to pass in a payer object (not using payees). `api-payee` was a value you could pass to use a `payee` as a `payer` we have not disabled this functionality, but thought it was really confusing so have just omitted its use here and in the docs for the time being. 

We then plan to add payers as its own resource via payees later :+1: